libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.html

```diff
@@ -6,9 +6,8 @@
         (click)="thumbnailClick(image)"
         (keydown.enter)="thumbnailClick(image)"
         tabindex="0"
-        role="button"
-        aria-pressed="false"
-        [title]="image.label"
+        [attr.title]="image.label"
+        [attr.ariaLabel]="image.label"
```

Comment:
Isn't it `attr.aria-label` ? Does this get read out by screen reader?

Reply:
yes its fixed now.